deploy/helm/events-collector-config.yaml

```diff
@@ -121,7 +121,8 @@ processors:
             - set(attributes["k8s.persistentvolume.name"], body["metadata"]["name"]) where body["kind"] == "PersistentVolume"
             - set(attributes["k8s.persistentvolumeclaim.name"], body["metadata"]["name"]) where body["kind"] == "PersistentVolumeClaim"
             - set(attributes["k8s.namespace.name"], body["metadata"]["namespace"])
-            - set(attributes["k8s.ingress.name"], body["metadata"]["name"]) where body["kind"] == "Ingress"
+            - set(attributes["k8s.ingress.name"], body["metadata"]["name"]) where body["kind"] == "Ingress" or body["kind"] == "VirtualService"
+            - set(attributes["k8s.ingress.type"], body["kind"]) where body["kind"] == "Ingress" or body["kind"] == "VirtualService"
```

Comment:
`k8s.ingress.type` is not an official OTEL k8s attribute. And there is a real chance that if it is added in the future, its meaning may be different from ours. We should call it `sw.k8s.ingress.type`.

Reply:
Sure, if it makes the attribute more futureproof 👍